integration/scenario_test.go

```diff
@@ -0,0 +1,830 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/dedis/d-voting/contracts/evoting/types"
+	"github.com/dedis/d-voting/internal/testing/fake"
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/suites"
+	"go.dedis.ch/kyber/v3/util/random"
+	"golang.org/x/xerrors"
+)
+
+var suite = suites.MustFind("Ed25519")
+
+// Check the shuffled votes versus the cast votes on a few nodes
+func TestScenario(t *testing.T) {
+	t.Run("Basic configuration", getScenarioTest())
+
+}
+
+func getScenarioTest() func(*testing.T) {
+	return func(t *testing.T) {
+		const (
+			loginEndpoint               = "/evoting/login"
+			createElectionEndpoint      = "/evoting/create"
+			openElectionEndpoint        = "/evoting/open"
+			castVoteEndpoint            = "/evoting/cast"
+			getAllElectionsIdsEndpoint  = "/evoting/allids"
+			getElectionInfoEndpoint     = "/evoting/info"
+			getAllElectionsInfoEndpoint = "/evoting/all"
+			closeElectionEndpoint       = "/evoting/close"
+			shuffleBallotsEndpoint      = "/evoting/shuffle"
+			beginDecryptionEndpoint     = "/evoting/beginDecryption"
+			combineSharesEndpoint       = "/evoting/combineShares"
+			getElectionResultEndpoint   = "/evoting/result"
+			cancelElectionEndpoint      = "/evoting/cancel"
+			initEndpoint                = "/evoting/dkg/init"
+		)
+
+		const contentType = "application/json"
+
+		t.Parallel()
+		proxyAddr1 := "http://localhost:8081"
+		proxyAddr2 := "http://localhost:8082"
+		proxyAddr3 := "http://localhost:8083"
+
+		proxyArray := [3]string{proxyAddr1, proxyAddr2, proxyAddr3}
+
+		// ###################################### CREATE SIMPLE ELECTION ######
+		create_election_js := `{"Configuration":{"MainTitle":"electionTitle","Scaffold":[{"ID":"YWE=","Title":"subject1","Order":null,"Subjects":null,"Selects":[{"ID":"YmI=","Title":"Select your favorite snacks","MaxN":3,"MinN":0,"Choices":["snickers","mars","vodka","babibel"]}],"Ranks":[],"Texts":null},{"ID":"ZGQ=","Title":"subject2","Order":null,"Subjects":null,"Selects":null,"Ranks":null,"Texts":[{"ID":"ZWU=","Title":"dissertation","MaxN":1,"MinN":1,"MaxLength":3,"Regex":"","Choices":["write yes in your language"]}]}]},"AdminID":"adminId"}`
```

Comment:
would be good to add some returns so that we can read the content of the configuration easily